util/containerutil/stub.go

```diff
@@ -7,12 +7,15 @@ import (
 	"github.com/pkg/errors"
 )
 
-// This is a stub for use in internal testing when its too much effort to provide a legitimate backend.
+// This is a stub for use when a proper frontend is not available.
 // Should never be used IRL.
```

Comment:
If we are going to use it IRL, then remove this comment

Reply:
Done